src/playercards/customizable/UpgradeSheetLibrary.ttslua

```diff
@@ -0,0 +1,364 @@
+-- static values
+Y_VISIBLE    = 0.25
+Y_INVISIBLE  = -0.5
+
+-- used for Summoned Servitor and Living Ink
+VECTOR_COLOR = {
+  unselected = { 0.5, 0.5, 0.5, 0.75 },
+  selected   = { 0.597, 0.195, 0.796 }
```

Comment:
I picked the purple color here because Ink was a mystic card, it won't be appropriate for others.  Let's key this by class (only need Mystic for now but at least have the option for the future)